src/components/Base/Header/Header.tsx

```diff
@@ -11,15 +11,25 @@ import {
   IconsContainer,
   Amount,
   IconContainer,
+  ThemeBtn,
 } from "./HeaderStyles";
 
-const Header = () => {
+const Header = (props: any) => {
   const cartContext = useContext(CartContext);
   const cartItemsAmount = cartContext.items.reduce((currentValue, item) => {
     return currentValue + item.amount;
   }, 0);
   return (
     <React.Fragment>
+      {props.theme === "dark" ? (
```

Comment:
it would be better to use the hook rather than passing the theme in the props 

```ts
import { useTheme } from 'styled-components'



const MyComponent = () => {

  const theme = useTheme()



  console.log('Current theme: ', theme)

  // ...

}
```